test/functional/provider/nodejs_spec.lua

```diff
@@ -0,0 +1,70 @@
+local helpers = require('test.functional.helpers')(after_each)
+local eq, clear = helpers.eq, helpers.clear
+local missing_provider = helpers.missing_provider
+local command = helpers.command
+local write_file = helpers.write_file
+local eval = helpers.eval
+local sleep = helpers.sleep
+local funcs = helpers.funcs
+local retry = helpers.retry
+
+do
+  clear()
+  if missing_provider('node') then
+    pending(
+      "Cannot find the neovim nodejs host. Try :checkhealth",
+      function() end)
+    return
+  end
+end
+
+before_each(function()
+  clear()
+end)
+
+describe('nodejs', function()
+  it('can inspect', function()
+    eq(1, funcs['provider#node#can_inspect']())
```

Comment:
This is going to fail on some developer systems, isn't it? It failed on mine...

Reply:
Yes, because it's meant for CI only and it's common to have old node versions if installing from OS package manager, not nvm.

Reply:
Ok, then why aren't the tests skipped on my system?

Reply:
Since Travis and Appveyor use nodejs 6.12+, it's pointless to have it now, even as a sanity check.